tests/Feature/Http/Controllers/Auth/AuthenticationTest.php

```diff
@@ -57,7 +57,7 @@ public function testUsersCanAuthenticateUsingTheLoginScreen(): void
     public function testUsersCanNotAuthenticateWithInvalidPassword(): void
     {
         // Given
-        $user = User::factory()->create();
+        $user = AuthorizedUser::factory()->create();
```

Comment:
Indien mogelijk, de variabele naam aanpassen naar $authorizedUser .

```suggestion
        $authorizedUser = AuthorizedUser::factory()->create();
````